hack/aks/Makefile

```diff
@@ -233,6 +233,28 @@ swift-up: rg-up swift-net-up ## Bring up a SWIFT AzCNI cluster
 		--yes
 	@$(MAKE) set-kubeconf
 
+swiftv2-multitenancy-cluster-up: rg-up
+	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
+		--network-plugin azure \
+		--network-plugin-mode overlay \
+		--kubernetes-version 1.28 \
+		--nodepool-name "mtapool" \
+		--node-vm-size Standard_D4_v2 \
```

Comment:
is this because of quota in region?

Reply:
I tried non-Standard_D4_v2 machine and it doesn't work.